README.md

```diff
@@ -42,6 +42,11 @@ POST {SQSD_HTTP_URL}\n
 <SQS message body>
 ```
 
+## Support 429 Status codes with Retry-After
+
+* SQSD will attempt to change the message visibility when the service responds with [429 status code](https://tools.ietf.org/html/rfc6585#section-4).
+* `Retry-After` response header can either contain an integer with the amount of senconds to wait or a [RFC1123](https://golang.org/pkg/time/#pkg-constants) formatted date.
```

Comment:
@jenriquejr why RFC1123? Do we need to support this or can we leave it as an int?